cmake/BuildLibCURL.cmake

```diff
@@ -211,7 +213,7 @@ macro(curl_build)
     list(APPEND LIBCURL_CMAKE_FLAGS "-DENABLE_CURLDEBUG=${TARANTOOL_DEBUG}")
     list(APPEND LIBCURL_CMAKE_FLAGS "-DENABLE_DEBUG=${TARANTOOL_DEBUG}")
     list(APPEND LIBCURL_CMAKE_FLAGS "-DUSE_MSH3=OFF")
-    list(APPEND LIBCURL_CMAKE_FLAGS "-DENABLE_WEBSOCKETS=OFF")
```

Comment:
These cmake options are not related to a fix in libcurl. Could you move these changes to a separate commit?

Reply:
These options are added or changed within the given range of curl's commits.

`cmake -LAH` difference:

```diff
--- curl-a.txt	2024-10-14 14:35:07.607066031 +0300
+++ curl-b.txt	2024-10-14 14:35:15.239066096 +0300
@@ -9,7 +9,7 @@
 BUILD_CURL_EXE:BOOL=ON
 
 // Build libcurl examples
-BUILD_EXAMPLES:BOOL=OFF
+BUILD_EXAMPLES:BOOL=ON
 
 // Build libcurl man pages
 BUILD_LIBCURL_DOCS:BOOL=ON
@@ -239,7 +239,7 @@
 // Path to the CA bundle has been set
 CURL_CA_BUNDLE_SET:BOOL=TRUE
 
-// Path to the CA bundle to embed into the curl tool.
+// Path to the CA bundle to embed in the curl tool.
 CURL_CA_EMBED:STRING=
 
 // Set ON to use built-in CA store of TLS backend. Defaults to OFF
@@ -311,6 +311,9 @@
 // Disable installation targets
 CURL_DISABLE_INSTALL:BOOL=OFF
 
+// Disable IPFS
+CURL_DISABLE_IPFS:BOOL=OFF
+
 // Disable Kerberos authentication
 CURL_DISABLE_KERBEROS_AUTH:BOOL=OFF
 
@@ -383,6 +386,9 @@
 // Disable verbose strings
 CURL_DISABLE_VERBOSE_STRINGS:BOOL=OFF
 
+// Disable WebSockets
+CURL_DISABLE_WEBSOCKETS:BOOL=OFF
+
 // Enable CMake export target
 CURL_ENABLE_EXPORT_TARGET:BOOL=ON
 
@@ -401,6 +407,9 @@
 // Enable compiler Link Time Optimizations
 CURL_LTO:BOOL=OFF
 
+// Bundle libtests/unittests into single binaries
+CURL_TEST_BUNDLES:BOOL=OFF
+
 // Enable BearSSL for SSL/TLS
 CURL_USE_BEARSSL:BOOL=OFF
 
@@ -473,9 +482,6 @@
 // Enable Unix domain sockets support
 ENABLE_UNIX_SOCKETS:BOOL=ON
 
-// Enable WebSockets (experimental)
-ENABLE_WEBSOCKETS:BOOL=OFF
-
 // Path to a program.
 HTTPD:FILEPATH=HTTPD-NOTFOUND
 
@@ -545,7 +551,7 @@
 // Enable librtmp from rtmpdump
 USE_LIBRTMP:BOOL=OFF
 
-// Use msquic library for HTTP/3 support
+// Use msh3/msquic library for HTTP/3 support
 USE_MSH3:BOOL=OFF
 
 // Use nghttp2 library
```

Reply:
> These options are added or changed within the given range of curl's commits.

Got it, thanks!